kalman_filter_passat:
    state_dimension : 10
    constant_system_matrix:
        - [1, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 1, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 1, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 1, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 1, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 1, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 1, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 1, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 1, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 1]

    time_variant_system_matrix:
        - [0, 0, 0, 1, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 1, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]

    time_variant_process_noise_matrix:
        - [0.1, 0, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0.1, 0, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0.1, 0, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0.1, 0, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0.1, 0, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0.1, 0, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0.1, 0, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0.1, 0, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0.1, 0]
        - [0, 0, 0, 0, 0, 0, 0, 0, 0, 0.1]
